tests/tests_geomstats/test_riemannian_mean_shift.py

```diff
@@ -16,14 +16,13 @@ class TestRiemannianMeanShift(geomstats.tests.TestCase):
     @geomstats.tests.np_autograd_and_torch_only
     def test_hypersphere_riemannian_mean_shift_predict(self):
```

Comment:
here we should remove `riemannian_mean_shift` from the name of this function: we know that we are in the riemannian_mean_shift test, from the name of the file: [test_riemannian_mean_shift.py](https://github.com/geomstats/geomstats/pull/1525/files#diff-5f6073f78a356fac372ab00b0bb5293e635899530eb4b37d647c43162519ae20)

@shubhamtalbar96 Could you update the names of all the tests accordingly? Thanks